fastlane/Fastfile

```diff
@@ -444,8 +444,11 @@ platform :ios do
   lane :build_and_upload_itc do |options|
     ensure_sentry_installed
 
-    ios_build_prechecks(skip_confirm: options[:skip_confirm], external: true) unless options[:skip_prechecks]
-    ios_build_preflight unless options[:skip_prechecks]
+    unless options[:skip_prechecks]
+      ios_build_prechecks(skip_confirm: options[:skip_confirm], external: true)
+      ios_build_preflight
+      xcversion() # Ensure we're using the right version of Xcode, defined in `.xcode-version` file
```

Comment:
Nitpick. Have you considered wrapping `xcversion()` in a function to integrate the comment in the code? Something like `check_xcode_version` or `enforce_xcode_version`?

Reply:
Update: I accidentally landed on [this document](https://github.com/xcpretty/xcode-install/blob/43328e923c682b7c0f30d806d03449e810738123/MIGRATION.md#migrating-fastlane-actions-that-depend-on-the-xcode-install-gem-to-use-xcodes) that mentions `xcode-install`/`xcversion` has been deprecated and how to update to the new `xcodes` alternative.

Might be worth looking into before merging, or as an update followup.

Reply:
> or as an update followup.

I'd be happy to look into it if your plate's full.

Reply:
Oh TIL this was deprecated. Yeah I'd love if you can take a look at migrating the command to `xcodes`. A bit unfortunate to see that it seems to require the installation of a separate tool tho, unlike `xcversion`…

Reply:
Seems latest fastlane shipped with official new usage of `xcodes` over `xcversion`: https://twitter.com/fastlanetools/status/1591561723794030592